VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.html

```diff
@@ -4,7 +4,20 @@
 
 <div id="participant-status-list">
   <div *ngFor="let participant of participants" class="govuk-grid-row">
-    <div class="govuk-grid-column-one-half">
+    <div class="govuk-grid-column-one-half status-box-display">
+      <div class="interpreter-icon">
+        <span *ngIf="participant.isInterpreterOrInterpretee" class="vh-interpreter-icon">
+          <em class="far fa-2x fa-language"></em>
+        </span>
+        <img
+          *ngIf="participant.isJudge"
+          src="/assets/images/govuk-crest.png"
+          height="40"
+          width="40"
+          class="govuk-header__logotype-crest"
+          alt="HM Courts &amp; Tribunals Service crest"
```

Comment:
`[attr.alt]="'participant-status.hmcts-crest' | translate"`
"hmcts-crest": "HM Courts & Tribunals Service crest" added to `en.json:717`

Reply:
added.